docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+cluster, the cluster topology is available, making the node protocol a good fit for use cases that use a relatively 
+small number of persistent connections.
+
+When you use the HTTP protocol, you can configure the Logstash Elasticsearch output plugin to automatically 
+load-balance indexing requests across a 
+{logstash}plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-host[specified set of hosts] in the 
+Elasticsearch cluster. Specifying multiple Elasticsearch nodes also provides high availability for the Elasticsearch 
+cluster by routing traffic to active Elasticsearch nodes.
+You can also use a third-party load balancer, either hardware or software, to handle connections between Logstash and 
+external applications.
+
+NOTE: Make sure that your Logstash configuration does not connect directly to Elasticsearch 
+{ref}modules-node.html[master nodes]. Master nodes tend to have substantial demands on their computing resources. 
+Connect to client or data nodes to maintain cluster performance.
+
+image::images/deploy_4.png[]
+
+// [float]
+[[deploying-message-queueing]]
+=== Managing Throughput Spikes with Message Queueing
+
+When a Logstash pipeline produces data faster than the Elasticsearch cluster can ingest, use a message queue to serve 
+as a buffer. By default, Logstash propagates backpressure to throttle incoming events when indexer consumption rates 
+fall below incoming data rates. Since this throttling can lead to events being buffered at the data source, preventing 
+backpressure with message queues becomes an important part of managing your deployment.
+
+Adding a message queue to your Logstash deployment also provides a level of protection from data loss. When a Logstash 
+instance that has consumed data from the message queue fails, the data can be replayed from the message queue to an 
+active Logstash instance.
+
+Several third-party message queues exist, such as Redis, Kafka, or RabbitMQ. Logstash provides input and output plugins 
+to integrate with several of these third-party message queues. When your Logstash deployment has a message queue 
+configured, Logstash functionally exists in two phases: shipping instances, which handles data ingestion and storage in 
+the message queue, and indexing instances, which retrieve the data from the message queue, apply any configured 
+filtering, and write the filtered data to an Elasticsearch index.
+
+image::images/deploy_5.png[]
+
+// [float]
+[[deploying-logstash-ha]]
+=== Multiple Connections for Logstash High Availability
+
+To make your Logstash setup more resilient to individual instance failures, you can set up a load balancer between your 
+data source machines and the Logstash cluster. The load balancer handles the individual connections to the Logstash 
+instances, ensuring that when an individual instance becomes unavailable, data ingestion and processing can continue.
```

Comment:
Also lets add that this architecture parallelizes the LS workload based on the inputs. If you add more inputs, you can spin up more LS instances horizontally